test-end-to-end/webapp-cypress/cypress/e2e/001_001_add_content.cy.js

```diff
@@ -0,0 +1,54 @@
+describe('001_001_add_content', () => {
+ it('Start Ustad Test Server ', () => {
+  // Start Test Server
+    cy.ustadStartTestServer()
+})
+
+it('Admin user add content to the library', () => {
+
+  // Admin user login
+  cy.ustadClearDbAndLogin('admin','testpass',{timeout:8000})
+  // Add H5p File
+  cy.ustadAddContentToLibrary('../test-files/content/H5p_Content.h5p','Content_001')
+  cy.contains('Content_001').click()
+  cy.contains('OPEN').click({force: true})
+  cy.get('#appbar_title').should('be.visible').invoke('text').should('eq', 'Content_001');
+
+ // Verify H5p Content
+ /*
+  https://www.lambdatest.com/blog/how-to-handle-iframes-in-cypress/
+  The iframe reference is from the above link
+ */
+  cy.get('iframe')
```

Comment:
I think the cy.get('iframe') isn't really do anything here because the return result is not used, it is just asserting that we have an iframe (which isn't needed given that we have the important assertions about the content of the iframe). If we want to make it clear - you can change line 23 to:
```
cy.get("iframe#xapi_content_frame")
```